stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -84,8 +87,8 @@ fn advance_to_nakamoto(peer: &mut TestPeer) {
 
         peer.tenure_with_txs(&txs, &mut peer_nonce);
     }
-
     // peer is at the start of cycle 8
+    peer.init_stacker_db_miners();
```

Comment:
Here, just call `peer.step_with_ibd(true)` after each call to `peer.tenure_with_txs(..)`.  That should initialize the `.miners` StackerDB.

Reply:
Hm it still breaks the make_stackerdb_config test. But perhaps I should just change that test D: